README.md

```diff
@@ -122,7 +123,8 @@ spec:
 
 | Name                                                | Description                                                                                       | Required | Default |
 | --------------------------------------------------- | ------------------------------------------------------------------------------------------------- | -------- | ------- |
-| `EPHEMERAL_ACCESS_DEFAULT_ROLE`                     | Defines the RoleName to be associated with users once the AccessRequest is created                | Yes      | -       |
+| `EPHEMERAL_ACCESS_DEFAULT_BASE_ROLE`                | Defines the default name shown as the current permission                                          | No       | -       |
```

Comment:
We need to clarify in the doc that this is just used to display and there is no association with a real role.

Reply:
Updated the readme to be more explicit that this is for display only